dbx/docker-replica-set.rst

```diff
@@ -1,4 +1,8 @@
-.. note:: Replica Set in Docker
+When a replica set runs in Docker, it might expose only one MongoDB endpoint.
+In this case, the replica set is not discoverable. Specifying ``directConnection=false``,
```

Comment:
S: I'd clarify what this option is on first mention (then removing "in your connection URI" the second time)
```suggestion
In this case, the replica set is not discoverable. Specifying ``directConnection=false`` in your connection URI,
```